SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -266,12 +276,10 @@ void CFlowOutput::SetAnalyzeSurface(CSolver *solver, CGeometry *geometry, CConfi
 
           Surface_StreamVelocity2[iMarker]   += Vn2*Weight;
           Surface_TransvVelocity2[iMarker]   += Vtang2*Weight;
-
+          
```

Comment:
Oh no problem on this one, that was just general for the future.
Just take care of this second blank space and feel free to merge.